Models/AutoEncoder3D.py

```diff
@@ -59,12 +62,14 @@ def validation_step(self, batch, batch_idx):
         loss = self.loss_fcn(prediction, image_dict[key])
 
         if batch_idx < 3:
-            im_st = image_dict[key][0, 0, 5, :, :]
-            plt.imshow(im_st.cpu())
+            im_st = image_dict[key][0, :, :, :, :]
+            self.logger.log_image(im_st, 'standard image', self.current_epoch)
+            plt.imshow(im_st[0, 5, :, :].cpu(), cmap='gray')
```

Comment:
Please don't leave comment like this in the code (imshow). The code online should not be for testing